chromium_src/content/browser/renderer_host/navigation_controller_impl.cc

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "content/browser/renderer_host/navigation_controller_impl.h"
+
+#include "brave/net/query_filter/query_filter.h"
+
+// This is for browser-initiated navigations (e.g. typing a URL in the URL bar).
+#define BRAVE_MAYBE_FILTER_TWITTER_PARAM                                      \
+  if (net::query_filter::ShouldFilterTwitterParam(url_to_load,                \
```

Comment:
nit: do we need to separate these into two functions? can we generalize this to be something like `net::query_filter::MaybeRewriteUrlQuery(params.initiator_origin, url_to_load);`?

note: mutable args go after const ones, that's why `params.initiator_origin` is first in this proposed function.

Reply:
Good point. No need for the two separate functions anymore.